### PR TITLE
Wind data + Data weather stations

### PR DESCRIPTION
**Pull request by Mapet13**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
 numpy==1.22.4
 netCDF4==1.5.8
-Pillow~=9.1.1
\ No newline at end of file
+pandas==2.0.1
+Pillow~=9.1.1
```
Comment (Herdran) [inline requirements.txt:4]: As per github dependabot alert we should change Pillow version to ~> 9.2.0

File: src/constatnts.py [hunk 2]
```diff
@@ -1,18 +1,50 @@
...
-CELL_SIDE_SIZE = 10  # points per cell side size
-GRID_SIDE_SIZE = 10  # cells per grid side size
-WORLD_SIDE_SIZE = CELL_SIDE_SIZE * GRID_SIDE_SIZE
+import pandas as pd
+from data.generic import Range
+from data.measurment_data import Coordinates
+from data.simulation_run_parameters import CellSideCount, SimulationRunParameters
+
+POINT_SIDE_SIZE = 50  # [m] #TODO: right now that could be different from what we have from coordinates :v - need to be calculated
+POINTS_SIDE_COUNT = 100
```
Comment (Herdran) [inline src/constatnts.py:7]: Does that mean we always assume square simulated space? Do we want it that way?

File: requirements.txt (hunk 1 above)
Comment (Herdran) [inline requirements.txt]: As per github dependabot alert we should change Pillow version to ~> 9.2.0

**Review by Herdran — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 1 above)
Comment (Mapet13) [reply requirements.txt]: ok

File: src/constatnts.py (hunk 2 above)
Comment (Mapet13) [reply src/constatnts.py:7]: i guess that was also done that way in previous version. If there will be time we could change that in a future

File: requirements.txt (hunk 1 above)
Comment (Mapet13) [reply requirements.txt]: Done

**Pull request merged**